README.md

```diff
@@ -82,8 +82,45 @@ By default the Elasticsearch service is available on: `http://localhost:9200`
 ## Azure Elasticsearch Setup
 To use this plugin first you will need to setup an Elaticsearch service.
 
-To user Microsoft Azure to provide an Elasticsearch service for Moodle:
-`<insert Azure instructions>`
+To use Microsoft Azure to provide an Elasticsearch service for Moodle:
+1. Create a Microsoft Azure account: [Account creation page] (https://azure.microsoft.com/en-in/free/)
```

Comment:
Watch the spaces between the brackets and the parentheses, it should be []() not [] ()